discord-bot/src/commands/quoteOfTheDay/index.test.ts

```diff
@@ -19,7 +24,24 @@ describe('Get quote of the day test', () => {
       channel: { send: replyMock },
       author: { bot: false },
     };
+    const fakeQuote = faker.lorem.words(25);
+    mockFetch.mockImplementationOnce(async () => ({
+      quote: fakeQuote,
+      author: 'Author',
+      html: `<h1>${fakeQuote}</h1>`,
+    }));
+    await getQuoteOfTheDay(mockMsg);
+    expect(replyMock).toHaveBeenCalledTimes(1);
+  });
+
+  it('Should return if no quote can be downloaded', async () => {
+    const mockMsg: any = {
+      content: `-qotd`,
+      channel: { send: replyMock },
+      author: { bot: false },
```

Comment:
And these two lines as well?

Reply:
Fixed in new commit, but instead of breaking this out, I just changed the format of all 1-attribute objects into a one liner if it doesn't break into a new line.